src/components/MailboxThread.vue

```diff
@@ -313,6 +332,47 @@ export default {
 		clearTimeout(this.startMailboxTimer)
 	},
 	methods: {
+		groupEnvelopesByDate(envelopes, syncTimestamp) {
+			const now = new Date(syncTimestamp)
+			const oneHourAgo = new Date(now.getTime() - 60 * 60 * 1000)
+			const startOfToday = new Date(now.getFullYear(), now.getMonth(), now.getDate())
+			const startOfYesterday = new Date(startOfToday)
+			startOfYesterday.setDate(startOfYesterday.getDate() - 1)
+			const startOfLastWeek = new Date(now)
+			startOfLastWeek.setDate(startOfLastWeek.getDate() - 7)
+			const startOfLastMonth = new Date(now)
+			startOfLastMonth.setMonth(startOfLastMonth.getMonth() - 1)
+
+			const groups = {
+				[t('mail', 'Last hour')]: [],
```

Comment:
I will suggest a different approach later because I find it a bit weird to use a translate string as key.